src/CargoBody.cpp

```diff
@@ -89,6 +102,7 @@ void CargoBody::TimeStepUpdate(const float timeStep)
 	if (m_hasSelfdestruct) {
 		m_selfdestructTimer -= timeStep;
 		if (m_selfdestructTimer <= 0) {
+			LuaEvent::Queue("onCargoDestroyed", this, NULL);
```

Comment:
Please prefer the use of `nullptr` over `NULL` everywhere in the code - NULL is not type-safe and doesn't follow our coding style. In addition, if you're trying to pass a `nil` parameter to Lua, simply leave the value off and only push one parameter - the LuaEvent mechanism works perfectly fine with 0..3 parameters in addition to the event name.